test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -203,7 +229,13 @@ describe('Switch Ethereum Chain for two dapps', function () {
           text: 'Confirm',
           tag: 'button',
         });
-        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+        await driver.clickElementAndWaitForWindowToClose({
+          text: 'Confirm',
+          tag: 'button',
+        });
+        await driver.delay(1000);
```

Comment:
you can use `await driver.delay(largeDelayMs);` instead here

Reply:
Can we poll for something here instead of the arbitrary delay?  We're trying to get rid of these.

Reply:
Sure! In the click above this we are waiting for the window to close after click. So, we can get rid of this delay. Updated here https://github.com/MetaMask/metamask-extension/pull/27980/commits/7f876c84993bb9649bb494b065bd0e8703034673